recipes/tz/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.files import get, copy, rmdir
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.53.0"
+
+class TzConan(ConanFile):
+    name = "tz"
+    license = "Unlicense"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.iana.org/time-zones"
+    description = "The Time Zone Database contains data that represent the history of local time for many representative locations around the globe."
+    topics = ("tz", "tzdb", "time", "zone", "date")
+    settings = "os", "build_type", "arch"
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
```

Comment:
```suggestion

    def build_requirements(self):
        self.tool_requires("mawk/1.3.4-20230404")

```

Add mawk to fix current build